clients/drcachesim/tracer/raw2trace.h

```diff
@@ -2087,6 +2110,10 @@ class raw2trace_t : public trace_converter_t<raw2trace_t> {
 
     // Chunking for seeking support in compressed files.
     uint64_t chunk_instr_count_ = 0;
+
+    // True if there was a prior branch delay. Used to indicate when to also
```

Comment:
I don't understand this logic: what does it mean that a branch was delayed at some point somewhere in the past?  Is this just trying to avoid the start-of-trace headers?  I would expect something specific to the start of the trace then and not a check for branches.  My proposal was to only delay if there is *currently* a delayed branch that has not been emitted yet, which seems the safest and clearest solution IMHO, avoiding delays we don't want regardless of where in the trace we are.

Reply:
Acknowledged. Updated to use the `delayed_branch` vector instead.